tests/Domain/Integrations/Controllers/IntegrationControllerTest.php

```diff
@@ -75,7 +194,7 @@ public function test_it_can_active_an_integration_with_an_organization(): void
             ]
         );
 
-        $response->assertRedirect('/nl/integraties/' . $integration->id->toString());
+        $response->assertRedirect("/nl/integraties/{$integration->id}");
 
         $this->assertDatabaseHas('organizations', [
```

Comment:
Now that I think about it, this assert is redundant and has no added value.

Reply:
Done